File: websocket/src/main/java/com/github/websocket/payload/Customer.java
package com.github.websocket.payload;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotBlank;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class Customer {

    private Long id;

    @NotBlank
    @JsonProperty(value = "customerName")
    private String customerName;

    @NotBlank
    @JsonProperty(value = "customerAddress")
    private String customerAddress;

    @NotBlank
    @JsonProperty(value = "customerEmail")
    private String customerEmail;

    @NotBlank
    @JsonProperty(value = "customerPhone")
    private String customerPhone;

}

File: server-mocks/src/main/kotlin/com/github/server/mocks/payload/Customer.kt
package com.github.server.mocks.payload

class Customer(
        val id: Long,
        val customerName: String,
        val customerAddress: String,
        val customerEmail: String,
        val customerPhone: String
) {

    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (javaClass != other?.javaClass) return false

        other as Customer

        if (id != other.id) return false
        if (customerName != other.customerName) return false
        if (customerAddress != other.customerAddress) return false
        if (customerEmail != other.customerEmail) return false
        if (customerPhone != other.customerPhone) return false

        return true
    }

    override fun hashCode(): Int {
        var result = id.hashCode()
        result = 31 * result + customerName.hashCode()
        result = 31 * result + customerAddress.hashCode()
        result = 31 * result + customerEmail.hashCode()
        result = 31 * result + customerPhone.hashCode()
        return result
    }

    override fun toString(): String {
        return "Customer(id=$id, customerName='$customerName', customerAddress='$customerAddress', customerEmail='$customerEmail', customerPhone='$customerPhone')"
    }

}